agent/tool-scripts/postprocess/openshift-metrics-postprocess

```diff
@@ -1,42 +1,35 @@
 #!/usr/bin/env python
-
-import sys, csv
+# -*- coding: utf-8 -*-
+import sys, csv, json
 
 input_file = sys.argv[1]
 output_file = sys.argv[2]
-in_csv = []
-metrics=[]
+metrics = []
 
-with open(input_file, 'r') as f:
-    reader = csv.reader(f)
-    for row in reader:
-        if len(row) == 3:
-          in_csv.append(row)
-metric_name = set()
-for row in in_csv:
-    metric_name.add(row[1])
-metric_names = list(metric_name)
-timestamp = set()
-for row in in_csv:
-    timestamp.add(row[0])
-timestamps = list(timestamp)
-header = 'timestamp'
-for name in metric_names:
-    header = header + ',' + name
-listlen = len(metric_names)
-for time_stamp in timestamps:
-    row = [None] * listlen
-    row.insert(0, time_stamp)
-    metrics.append(row)
-for row in in_csv:
-    timestamp_index = [i for i,x in enumerate(timestamps) if x==row[0]]
-    metric_index = [i for i,x in enumerate(metric_names) if x==row[1]]
-    if len(row) == 3:
-       metrics[timestamp_index[0]][metric_index[0]+1] = row[2]
-#output to a file
-file = open(output_file, 'w')
-file.write(header + '\n')
-wr = csv.writer(file)
-for line in metrics:
-    wr.writerow(line)
-file.close()
+in_file = open(input_file, "r")
+metrics_datalog = in_file.readlines()
+with open(output_file, 'w') as f:
+    for item in metrics_datalog:
+        values = []
+        try:
+          json.loads(item)
+        except ValueError:
+          continue
+        item = json.loads(item)
+        for value in item['metrics']:
+            values.append(value)
+        if len(values) == 1:
+            item[item['name']] = values[0].values()[0]
```

Comment:
Why did we have to use `values()` method of the dictionary object?  Why couldn't we use a reference to the key name itself?